migrations/sql/00000009_add_dataset_names.up.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE evaluation ADD COLUMN dataset_names jsonb;
+
+ALTER TABLE evaluation
+DROP CONSTRAINT evaluation_model_name_datum_filter_parameters_key;
+
+UPDATE evaluation
+SET dataset_names = datum_filter->'dataset_names';
+
+UPDATE evaluation
+SET datum_filter = jsonb_set(datum_filter, '{dataset_names}', 'null'::jsonb);
+
+ALTER TABLE evaluation ADD CONSTRAINT evaluation_unique_constraint
+UNIQUE (model_name, datum_filter, parameters, dataset_names);
```

Comment:
should `datum_filter` be renamed to `filter`? does it only ever filter datums?

Reply:
`filter` is a keyword in SQL so we can't use it directly. Would `filters` work for you?

Reply:
This could also be applied to the schemas

Reply:
filters works for me, thanks!